app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
@xprazak2 ,

You mean that there are some columns of rows which may have ONLY `result-filter-tag ` CSS class but not `label-{tag}` classes i.e label-info, label-warning and 
you want to show only those rows in which first column is having both classes `label-{tag}` and `result-filter-tag `.

That is the case then `$('.label-{tag}.result-filter-tag')` is needed and no modification which suggested by me in code required.